src/backend/request.rs

```diff
@@ -39,9 +39,10 @@ impl Request {
         #[cfg(feature = "tracing")]
         tracing::debug!("{_method}");
         let (fut, abort_handle) = abortable(callback);
+        let path_clone = path.clone();
```

Comment:
I don't really think the path must be passed to every function there is, i am kinda wondering if we should instead pass the AbortHandle...but as it doesn't implement hash, you would not be able to construct a mapping between spawned dialogs and the corresponding requests.

Reply:
My reasoning for path is that - it is really simple solution this way and we can get the path from incoming D-Bus request itself without having to store any state in the library side.

Also whatever we are passing for the interface request (like `Account.GetUserInformation`) we need to pass to `Request.close` too so as to establish the association.

Happy to try better ideas and workarounds :)

Reply:
The thing is I really don't want to expose the Request object path, if we really have to expose part of it, I would expose a HandleToken which we can retrieve from the last portion of the ObjectPath. Basically `HandleToken::try_from(path.as_str().split('/').last().unwrap()).unwrap()` or so.

Where are you using the backend feature btw?

Reply:
I think the `HandleToken`s also suit here. Do you have any specific suggestions on how we can expose it to the impls? May be we add a new constructor to `HandleToken` which lets us construct from an `ObjectPath` or so?

We are trying to rewrite Phosh portal in Rust and that's where we are using `ashpd` :).

Reply:
Interesting! let me know how the experiment goes. 

I would do a TryFrom<ObjectPath> for HandleToken that is gated by the backend feature.